index.html

```diff
@@ -942,7 +948,10 @@ <h3>
           {{PerformanceResourceTiming/encodedBodySize}}, and
           {{PerformanceResourceTiming/decodedBodySize}}. Further, the
           {{PerformanceResourceTiming/nextHopProtocol}} attribute will be set
-          to the empty string.
+          to the empty string. In addition, if the <a data-cite=
+          "FETCH#concept-cors-check"> CORS check</a> fails the following
```

Comment:
I think we need to enforce this in the processing model as well. (e.g. in the getter above)

Reply:
Do we still need to enforce this in the `getter`, since https://github.com/whatwg/fetch/pull/1468#discussion_r931266882 notes it's already enforced when we recieve it from fetch?

Reply:
That comment makes sense! Can you add a note indicating that info next to the getter? Thanks! :)

Reply:
Added